docs/en/02_Developer_Guides/00_Model/02_Relations.md

```diff
@@ -39,6 +39,15 @@ class Player extends DataObject
 This defines a relationship called `Team` which links to a `Team` class. The `ORM` handles navigating the relationship
 and provides a short syntax for accessing the related object.
 
+To create a has_one/has_many relationship to core classes (File, Image, etc), reference the Classname::class, like below.
+
+```php
+    private static $has_many = [
```

Comment:
In our docs we usually add the import statement at the top of the example block here.

So please add the FQCN for `Image` and `File` in `use` statements before this line